### PR TITLE
Feat: wishlist page & cart page

### PR DESCRIPTION
**Pull request by bhtibrewal**

### CODE REVIEW

File: pages/cart/cart.css [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    justify-content:space-around;
+    align-items: flex-start;
+    margin: 2rem 0;
+    width:100%;
+    padding:0 2rem;
+    gap:2rem;
+}
+.cart-sec {
+    flex-grow:1;
+  padding: 2rem 5%;
```
Comment (parvez-rashid) [inline pages/cart/cart.css:12]: I'm not sure if this is good practice; unless you have a particular purpose here, stick to consistent units.

File: style.css [hunk 2]
```diff
@@ -6,9 +6,13 @@ body {
     font-size: 30%;
   }
 }
-
-::-webkit-scrollbar {
-  display: none;
+.main::-webkit-scrollbar {
```
Comment (parvez-rashid) [inline style.css:14]: You don't need to give browser-specific (`webkit`) styling as it is handled automatically.

File: style.css [hunk 3]
```diff
@@ -6,9 +6,13 @@ body {
...
+  width:1.5rem;
+  padding:1rem;
+}
+.main::-webkit-scrollbar-thumb{
```
Comment (parvez-rashid) [inline style.css]: You don't need to give browser-specific (`webkit`) styling as it is handled automatically.

File: style.css [hunk 4]
```diff
@@ -6,9 +6,13 @@ body {
     font-size: 30%;
   }
 }
-
-::-webkit-scrollbar {
```
Comment (parvez-rashid) [inline style.css:10]: You don't need to give browser-specific (`webkit`) styling as it is handled automatically.

**Review by parvez-rashid — APPROVED** [review]
I have left comments below, have a look. Other than that, code and the preview looks great.

Edit: UI Review - The UI looks amazing, the opening modal is spot-on. Great work!

**Review by sri-rishi — CHANGES_REQUESTED** [review]
Screenshot (28)


Here in wishlist page button should have remove from wishlist. And just a suggestion comments are not necessary remove them before pushing.

**Review by sri-rishi — COMMENTED** [review]
Otherwise code looks fine to me

File: pages/cart/cart.css [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+    border-top:1px solid var(--primary-color)
+}
+.part-1, .part-2{
+    display:grid;
+    grid-template-columns: 1fr auto;
+    justify-items: stretch;
+}
+.part-1>*, .part-2>*{
+    line-height:3rem;
+}
```
Comment (pankaj-prog) [inline pages/cart/cart.css]: you can improve names here, part1 and part2 is not declarative

**Review by pankaj-prog — APPROVED** [review]
UI looks good.

File: pages/cart/cart.css (hunk 5 above)
Comment (bhtibrewal) [reply pages/cart/cart.css]: did the changes

**Pull request merged**